business_rules/engine.py

```diff
@@ -107,3 +128,17 @@ def fallback(*args, **kwargs):
         params = action.get('params') or {}
         method = getattr(defined_actions, method_name, fallback)
         method(**params)
+
+
+async def async_do_actions(actions, defined_actions):
+    for action in actions:
+        method_name = action['name']
+
+        def fallback(*args, **kwargs):
```

Comment:
```suggestion
        async def fallback(*args, **kwargs):
```

I think you should make `fallback` async in case the `method` is `fallback`